misc/semver-checker/modules/semver-checker-core/src/test/java/org/ballerinalang/semver/checker/ProjectUtil.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.semver.checker;
+
+import io.ballerina.projects.BuildOptions;
+import io.ballerina.projects.directory.BuildProject;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.StringJoiner;
+
+/**
+ * This is the class for getting the ballerina package instance from the text code.
+ */
```

Comment:
add `@since 2201.2.0` tag. (for all the new java classes)

Reply:
Added